CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
-*
+* word breaking in `alert` component
```

Comment:
Capitalize + try making this more clear. The point with the changelog is: a non-tech person should be able to understand it.

Example: Added word breaking to the `alert` component, so that the entire text is always visible